en_US/changes/changes-4.3.md

```diff
@@ -15,6 +15,91 @@ ref: undefined
 
 # Changes
 
+## 4.3.0 版本
+
+*Release Date: 2021-05-06*
+
+EMQ X 4.3.0 is released now, it mainly includes the following changes:
+
+### Features and Enhancement
+
+#### Building
+
+- Support Erlang/OTP 23
+- The new installation package only supports macOS 10.14 and above
+- Project adjusted to umbrella structure
+
+#### Performance improvement
+
+- The underlying implementation of the multi-language extension function is changed from erlport to gRPC
+- Support routing table compression, reduce memory usage, enhance subscription performance, publishing performance will be slightly affected, so disable option is provided
+- Improve wildcard subscription performance
+- Improve processing performance when a large number of clients are offline
+
+#### Safety
+
+- Protect EMQ X Broker from cross-site WebSocket hijacking attacks
+- SSL supports `verify` and `server_name_indication` configuration
+- Support the configuration of the maximum length of the certificate chain and the password of the private key file
+
+#### Other
+
+- Added update resource logic for rule engine
+- Rule engine SQL function supports conversion between unix timestamp and rfc3339 format time
+- Keep retrying the resources that failed to connect after the EMQ X Broker is started
+- Websocket listener supports selecting supported subprotocols from the subprotocols list
+- WebSocket connection supports obtaining real IP and Port
+- Support the default authentication method caching_sha2_password of MySQL 8.0
+- JWT authentication supports JWKS
+- The starting point is randomly selected when the shared subscription distribution strategy is configured as `round_robin`
+- Shared subscription supports Hash distribution of messages by source topic
+- Support import and export of Authentication & ACL information in Mnesia
+- Allow to use Base64 encoded client certificate or MD5 value of client certificate as username or Client ID
+- Support restarting the listener
+- Only enable data telemetry in the official version
+- Support clearing all ACL cache
+- Support observer_cli
+- Quickly kill the connection process when OOM
+- Support cluster metrics for Prometheus
+- Redis sentinel mode supports SSL connection
+- Support single-line log output, and support rfc3339 time format
+
```

Comment:
Please add: environment variable override for configuration is made available for all distributions (previously only for docker)